README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Hi!
+
+My name is **Ojaswi Malik**. 
+
+![My Image](Ojaswi.jpeg) 
```

Comment:
Nice relative path! Your pull request is separate from this repo though, so I believe when you tried to link this in that text box, it did not render. You can also use URLs to link in that circumstance.

Reply:
I like Ojaswi's bio because it's quite clear and straightforward in terms of her personal and professional interests. Also, she considered all the requested elements for homework.